client/package.json

```diff
@@ -23,7 +23,8 @@
     "web-vitals": "^1.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "REACT_APP_CTX=docker react-scripts start",
+    "start2": "REACT_APP_CTX=native react-scripts start",
```

Comment:
Let's rename this to something more descriptive. `start2` isn't really telling me how/why it's different. Something like `start-native` would be more descriptive.